src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/services/GetRiskSeriousHarmForPersonService.kt

```diff
@@ -19,6 +20,9 @@ class GetRiskSeriousHarmForPersonService(
 
     if (deliusCrn != null) {
       personRisks = assessRisksAndNeedsGateway.getRiskSeriousHarmForPerson(id = deliusCrn)
+      if (personRisks.hasError(UpstreamApiError.Type.FORBIDDEN)) {
```

Comment:
If we refactor AssessRisksAndNeedsGateway to return emptyList() instead of null, then we will not need to have this if statement any more